hello-data-portal/hello-data-portal-api/src/main/java/ch/bedag/dap/hellodata/portal/metainfo/service/MetaInfoResourceService.java

```diff
@@ -123,4 +125,34 @@ public String findInstanceUrl(ModuleType moduleType, String instanceName) {
         }
         return null;
     }
+
+    @Transactional(readOnly = true)
+    public String findSupersetInstanceNameByContextKey(String contextKey) {
+        return findAllByModuleTypeAndKind(ModuleType.SUPERSET, ModuleResourceKind.HELLO_DATA_APP_INFO,
+                                          ch.bedag.dap.hellodata.commons.sidecars.resources.v1.appinfo.AppInfoResource.class).stream()
```

Comment:
Do we have multiple Classes with same name?

Reply:
no, why?

Reply:
ah, I though so because of the fully qualified class-name.